src/api/app/views/webui/package/show.html.haml

```diff
@@ -1,10 +1,22 @@
+-# haml-lint:disable ViewLength
```

Comment:
Used to avoid 
`app/views/webui/package/show.html.haml:0 [W] ViewLength: View template is too long [107/100]`